### DISCUSSION TITLE
Question on abuse of multiboxer

### DISCUSSION THREAD

**Discussion by finalmortal** [chromiecraft/chromiecraft#4450] [General]
Multiboxer *redacted* joined a rfd run and didn't play with the group and then kicked people and forced a player to have a 30 minute deserter buff. 

Seems like it would be some kind of violation and i didn't see a place to post about questions like this right off hand.

**Comment by 55Honey** [comment]
Hello @finalmortal 
Please refrain from public call-outs. We offer a ticketing system for reports on our discord:
https://discord.gg/ChromieCraft

For us to investigate, please open a ticket and provide footage or some other kind of proof of the incident.

Thanks!
